src/transport/types.py

```diff
@@ -1,3 +1,4 @@
 class TransportType:
     RABBIT = 'rabbit'
     KAFKA = 'kafka'
+    DATA_BUS = 'data_bus'
```

Comment:
Data bus is just a abstract transport name for DSM - kafka and rabbit also are data buses.
Let's name it `l2_transport`

Reply:
Renamed